dev-aws/kafka-shared-msk/energy-budget-plan/energy-budget-plan.tf

```diff
@@ -68,6 +68,24 @@ resource "kafka_topic" "fabricator_deadletter" {
   }
 }
 
+resource "kafka_topic" "assessment_request" {
+  name           = "energy-budget-plan.assessment-request"
+  replica_factor = 3
```

Comment:
This needs to be `replication_factor`. The applier is failing due to this: https://terraform-applier-system.dev.aws.uw.systems/#pubsub-kafka-shared-msk-energy-budget-plan